app/src/main/java/org/sopt/dosopttemplate/DoAndroidFragment.kt

```diff
@@ -0,0 +1,33 @@
+package org.sopt.dosopttemplate
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sopt.dosopttemplate.databinding.FragmentDoAndroidBinding
+
+class DoAndroidFragment: Fragment() {
+    private var _binding : FragmentDoAndroidBinding? = null
+    private val binding: FragmentDoAndroidBinding
+        get() = requireNotNull(_binding){ "바인딩에러" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentDoAndroidBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
```

Comment:
```suggestion
    override fun onDestroyView() {
        _binding = null
        super.onDestroyView()
    }
```
요렇게 하면 _binding 변수가 먼저 해제되므로 잠재적인 메모리 누수를 방지할 수 있어서 안전하게 변수를 null 처리 해줄 수 있슴당!

Reply:
맞아요 이래야 안정성이 증가한다구 librarywon님한테 배웠어요

Reply:
짱짱.. 이번과제할때 다 수정해야겟다 꼼꼼한 리ㅣ뷰 고마워 ~!!!!